File: Java/PB with Java/src/CalculationOfTheAreaOfARectangle.java
import java.util.Scanner;

public class CalculationOfTheAreaOfARectangle {
    public static void main(String[] args) {
        double sideA, sideB;
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter side a: ");
        sideA = Double.parseDouble(scanner.nextLine());
        System.out.print("Enter side b: ");
        sideB = Double.parseDouble(scanner.nextLine());
        double area = sideA * sideB;
        System.out.println("Area: " + area);
    }
}

File: C++/Programming Basics with C++/First Steps In Programming - Lab and Exc/CalculationOfTheAreaOfARectangle/CalculationOfTheAreaOfARectangle.cpp
#include <iostream>

int main() {
    double sideA, sideB;
    std::cout << "Enter side a: ";
    std::cin >> sideA;
    std::cout << "Enter side b: ";
    std::cin >> sideB;
    double area = sideA * sideB;
    std::cout << "Area: " << area << std::endl;

    return 0;
}
